### DISCUSSION TITLE
Please make Thorium reader available in Flathub

### DISCUSSION THREAD

**Discussion by mbridon** [edrlab/thorium-reader#2481] [General]
[Flathub](https://flathub.org) is the new Linux app store, working on all Linux distributions and delivering apps that are secure and easy to install.

The security and ease of use comes from Flatpak integration, apps are isolated in sandboxes so they can't access the private user data without their explicit interaction.

Apps on Flathub are maintained by a community of foss volunteers along with the developers of the app themselves.

Being a Flathub member, I can help if needed. :wink:

**Comment by danielweck** [comment]
duplicate https://github.com/edrlab/thorium-reader/discussions/1750
